src/smt/smt_setup.cpp

```diff
@@ -28,13 +28,15 @@ Revision History:
 #include "smt/theory_array_full.h"
 #include "smt/theory_bv.h"
 #include "smt/theory_datatype.h"
+#include "smt/theory_recfun.h"
 #include "smt/theory_dummy.h"
 #include "smt/theory_dl.h"
 #include "smt/theory_seq_empty.h"
 #include "smt/theory_seq.h"
 #include "smt/theory_pb.h"
 #include "smt/theory_fpa.h"
 #include "smt/theory_str.h"
+#include "ast/recfun_decl_plugin.h" // TODO: instead, theory_recfuns
```

Comment:
theory_recfun.h could just include ast/recfun_decl_plugin.h so you don't have to import it here too.